easybuild/easyblocks/h/healpix.py

```diff
@@ -62,11 +72,32 @@ def configure_step(self):
         if not cfitsio:
             raise EasyBuildError("Failed to determine root for CFITSIO, module not loaded?")
 
+        # target:
+        #   1: basic_gcc
+        #   2: generic_gcc
+        #   3: linux_icc
+        #   4: optimized_gcc
+        #   5: osx
+        #   6: osx_icc
+
         self.comp_fam = self.toolchain.comp_family()
         if self.comp_fam == toolchain.INTELCOMP:  # @UndefinedVariable
-            cxx_config = '4'  # linux_icc
+            cxx_config = '3'  # linux_icc
```

Comment:
the changing number here is a sign of trouble (changing number for the same target across versions)

I've change things in https://github.com/jose-d/easybuild-easyblocks/pull/2 such that you don't need to hardcode the numbers anymore, you can extract it from the question (like we do in the WRF easyblock).

Works for both `HEALPix-2.20a-ictce-5.5.0.eb` and `HEALPix-3.50-foss-2018b.eb`

Reply:
good job.